### ISSUE TITLE
(Self-voicing) - are directional strings needed with mouse interaction

### ISSUE THREAD

**Issue by terracoda** [phetsims/gravity-force-lab-basics#211]
Noticing with mouse interaction, I only get the orientational cues like , “Closer” and “Farther away” if I move a tiny bit using the mouse interaction. 

Leave as-is for now. I am wondering if others will notice and if they find it ok, or strange?

**Comment by terracoda** [comment]
There was feedback from the Team Survey that they wanted to hear "Closer" and "Farther" rather than a described distance (very close to, very far from, etc.)

@jessegreenberg, is it possible to incorporate some of the progress responses into a mouse interaction? Could we increase the distance threshold for the "Closer"/"Farther" responses fire? 

Would this cause issues for touch. The same thresholds could apply to touch as well as mouse.

**Comment by jessegreenberg** [comment]
Discussed today, with mouse and touch input we should hear 'closer'/'farther' on end of drag after changing position of mass, just like form keyboard input.

**Comment by jessegreenberg** [comment]
@terracoda with mouse interaction, should we also be getting things like "Track end right" and other boundary things? These are coming through with keyboard and I can't remember if it was an explicit design choice to not have these as well when using a mouse.

**Comment by jessegreenberg** [comment]
From the design document:
> Question for Meeting: We could consider dropping orientational cues which were specifically added for non-visual access (e.g. “right side of track” and “Closer”, “Last stop right”) to make them slightly shorter.

**Comment by jessegreenberg** [comment]
OK, the reason it isn't coming through on mouse input is because of the original implementation. this.forceDescriber.getVectorChangeText which includes the direction change assmumes discrete motion with a keyboard and motion with a mouse is registered too quickly - the function thinks that the mass has been moving in the same direction every time it is called because the mass positions change faster than the description is generated.

**Comment by jessegreenberg** [comment]
The function uses a flag `this.lastMovedCloser` which is updated every mass position change. I wonder if it would work better if it were updated when drag ends? Objects have a `dragging` field. Could that be turned into a Property to update this.lastMovedCloser instead?

**Comment by terracoda** [comment]
Re:
**Question for Meeting:** We could consider dropping orientation cues which were specifically added for non-visual access (e.g. “right side of track” and “Closer”, “Last stop right”) to make them slightly shorter.

I think I asked that question to myself before I got the feedback from the Team Survey.

After the team survey, I feel that these cues may be very good for mouse users as well.

**Comment by jessegreenberg** [comment]
Thanks @terracoda - I also noticed that question was in a Paradigm 1 section, so may not be related anymore.

**Comment by terracoda** [comment]
Yes, I think that question is not relevant anymore.

**Comment by jessegreenberg** [comment]
> with mouse interaction, should we also be getting things like "Track end right" and other boundary things?

Over slack, @terracoda  mentioned that these ARE coming through with mouse input, so this is not an issue.

**Comment by terracoda** [comment]
@jessegreenberg, it's only the progress phrases "Closer" and "Farther away" that seem to be missing with mouse interaction.  You can work on that as you can. I don't expect that for tomorrow.

**Comment by terracoda** [comment]
@jessegreenberg, I hear the "closer" and "farther away" when using a mouse if I move the sphere a tiny bit (likely the same size as a default key press).

In the original design I think these phrases are meant for changes in direction.

Is it possible, to make that also true for Mouse interaction, or is it possible to make the threshold for it it to fire with a mouse or touch event a little wider?

**Comment by terracoda** [comment]
@jessegreenberg, any thoughts on feasibility of making the threshold for the progeess reponse a little wider for mouse/touch?

**Comment by jessegreenberg** [comment]
In #251 we added directional content during change with mouse/touch. This can be closed.
